### PR TITLE
Merges 1.3.8&1.3.9 release into master

### PR DESCRIPTION
**Pull request by ashfurrow**
See discussion in https://github.com/artsy/emission/pull/612

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ### Master
 
+
+### 1.3.8
+
+- [dev] Moves native view controllers to use iOS7+ status bar api - ash
+
```
Comment (orta) [inline CHANGELOG.md]: this feels wrong?

Comment (orta) [inline CHANGELOG.md]: ah, needs a master merge - makes sense

Comment (ashfurrow) [inline CHANGELOG.md]: Yup, I'm resolving things locally first. I'll un-WIP the PR once it's ready.

**Comment by ashfurrow** [comment]
Okay, should be good to review 👍

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Changes were made to package.json, but not to yarn.lock - <i>Perhaps you need to run `yarn install`?</i>
  </td>
    </tr>
  </tbody>
</table>


:tada:
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**